### ISSUE TITLE
stability: unhelpful errors from `freeze-cluster`

### ISSUE THREAD

**Issue by bdarnell** [cockroachdb/cockroach#7805]
Running `cockroach freeze-cluster` with no parameters results in the following cryptic message.

```
ubuntu@cockroach-beta-4:~$ ./cockroach version
Build Tag:   beta-20160629-165-g30175ad
Build Time:  2016/07/07 19:19:44
Platform:    linux amd64
Go Version:  go1.6.2
C Compiler:  gcc 4.9.2
ubuntu@cockroach-beta-4:~$ ./cockroach freeze-cluster 
Error: rpc error: code = 13 desc = transport is closing
Failed running "freeze-cluster"
```

Passing `--host` without the certificate options for a secure cluster gives:
```
ubuntu@cockroach-beta-4:~$ ./cockroach freeze-cluster --host localhost
Error: rpc error: code = 13 desc = transport: write tcp 127.0.0.1:33590->127.0.0.1:26257: write: broken pipe
Failed running "freeze-cluster"
```

In both cases we should do a better job of diagnosing and reporting what went wrong. This probably isn't unique to `freeze-cluster`, but since this is going to be part of our upgrade process for a future release it's likely to be one of the first commands an admin runs that connects over grpc instead of pgwire.

**Comment by tamird** [comment]
Should this have been closed by #9562? @tschottdorf

**Comment by tschottdorf** [comment]
We really need to start using freeze-cluster again in production, but yes I think so.
